src/08-accessing.js

```diff
@@ -148,7 +164,9 @@ function getCityAndState(person) {}
  *  getFullAddress(person);
  *  //> "697 Pine Drive 2A, Staten Island, NY, 10306"
  */
-function getFullAddress(person) {}
+function getFullAddress(person) {
+  return person.address.street + " " + person.address.unit + "," + " " + person.address.city + "," + " " + person.address.state + "," + " " + person.address.zip
```

Comment:
This is a great example of why we have template literals.  Remember that with backticks we can spread out a string of many lines.  Moving this string down over multiple lines in this case would make it much easier to read.